devtools/dev-requirements.txt

```diff
@@ -1,6 +1,9 @@
 # standard install requirements
 -r ../requirements.txt
 
+# to use jupyter notebooks
+ipykernel
```

Comment:
Are you sure you want to add this? `ipykernel` may be installed in different ways depending on the user's development setup. Sometimes its managed separately through an older package `nb_conda_kernels` or the package manager `sudo dnf install jupyterlab`.

Reply:
Tbh, I don't know why this happens, but sometimes `ipykernel` is not installed with `pip install -r devtools/dev-requirements.txt`. However, most of the time it is installed without needing to add this line. When it is not installed VS-Code complain for notebooks using `desc-env`, and clusters don't even show the `desc-env` for interactive sessions (unless when you start you change this option 
![image](https://github.com/user-attachments/assets/21dae26d-7dd2-4b52-b956-ee9ca7537e12)
)
I don't think this will cause any issue for other users. I added this just to prevent this kind of rare instances.